modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-production-deployment.adoc

```diff
@@ -65,14 +65,28 @@ The Redpanda Helm chart enables TLS by default and uses cert-manager to manage T
 +
 include::deploy:partial$kubernetes/install-crds.adoc[]
 
-. Create a YAML file to override the default version of the Redpanda Operator.
+. [[config-operator]]Create a YAML file to override the default version of the Redpanda Operator.
 +
 .`redpanda-operator-values.yaml`
 [,yaml,subs="attributes+"]
 ----
 image:
   tag: {latest-operator-version}
 ----
++
+[NOTE]
+=====
+If you already have Flux installed and you want it to continue managing resources across the entire cluster, use the following:
+
+.`redpanda-operator-values.yaml`
+[,yaml]
+----
+additionalCmdFlags:
+  - --enable-helm-controllers=false
+----
+
+This flag prevents the Redpanda Operator from deploying its own set of Flux controllers that may conflict with existing ones.
```

Comment:
We may want to add a warning here :/ While that flag does disable the flux controllers that are bundled into the operator it, we still bundle a modified version of Flux's CRDs in with the redpanda CRDs which can break an existing flux installation if the versions aren't exactly the same.

Reply:
> which can break an existing flux installation if the versions aren't exactly the same.

@chrisseto please could you clarify this part? Which versions of what software must be exactly the same?

Reply:
The version of flux installed in the enduser's cluster must exactly match the version of flux that the operator vendors. And even then, it's quite possible that it doesn't work because we're slightly customizing the flux CRDs :[

Reply:
What's the recommended way to find out the version of Flux that comes installed with our operator?

Reply:
Unfortunately, that's defined in two places right now. Thankfully they're the same right now.

Once in the [Makefile](https://github.com/redpanda-data/redpanda-operator/blob/main/src/go/k8s/Makefile#L7-L8) (The version of the flux CRDs) and once in [go.mod](https://github.com/redpanda-data/redpanda-operator/blob/main/src/go/k8s/go.mod#L12-L17) (The version of the flux controller that's actually run by the operator)

Reply:
I've added a warning. Let me know if that works.